dataprofiler/data_readers/filepath_or_buffer.py

```diff
@@ -1,7 +1,21 @@
-from io import open, StringIO, BytesIO, TextIOWrapper
+from io import BytesIO, StringIO, TextIOWrapper, open
```

Comment:
I think you need to remove the import below for the data_utils

Reply:
👍